tox.ini

```diff
@@ -15,13 +16,13 @@ deps=
     # Newer versions of gwcs require astropy 4.x, which
     # isn't compatible with the older versions of numpy
     # that we test with.
+    legacy: astropy~=5.0.4
     legacy: gwcs==0.9.1
-    legacy: semantic_version==2.8
-    legacy: pyyaml==3.13
     legacy: jsonschema==4.0.1
-    legacy: numpy~=1.14.6
-    legacy: pytest~=4.6.11
-    legacy: astropy~=3.0.0
+    legacy: numpy~=1.18
+    legacy: pytest~=6.0.0
+    legacy: pyyaml==3.13
+    legacy: semantic_version==2.8
```

Comment:
Why have these been reordered? It makes following the diffs really hard

Reply:
I ordered these alphabetically, as that's the standard I've seen in other projects. I can revert this change though, you're right it makes it difficult to track